internal/driver/registry_default.go

```diff
@@ -219,6 +222,11 @@ func (r *RegistryDefault) ReadRouter() http.Handler {
 	}
 
 	n.UseHandler(br)
+
+	if r.sqaService != nil {
```

Comment:
Is this ever nil? Can we ensure that this doesn't accidentally not run because we forgot to initialize it?

Reply:
It is nil if we do not initialize it.
The problem is, unlike the other projects, we start the server in keto in tests etc. outside of a `cobra.Command` context. That is why I did it this way. There are valid cases where the `sqaService` cannot be initialized.

Reply:
Ok, can we add a safe guard to ensure this works when not run using the tests? E.g. via context or something?

Reply:
What about a `func (r *Registy) ServeWithSQA()` that first adds the service and then serves in one go?

Reply:
SGTM - but keep in mind that you need the cmd passed down to metricsx.New